libsolidity/ast/Types.cpp

```diff
@@ -190,6 +190,9 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 
 	for (auto const& type: _parameters)
 	{
+		if (!type)
```

Comment:
Wouldn't it be better to push nullptr instead?

Reply:
I was trying to reproduce the problem that initially caused me to do this but it seems it's gone now, probably due to a rebase. So I can safely remove this change.

Reply:
I added this to fix a segfault but I cannot reproduce this problem anymore, so now I removed this (and similar fixes)

Reply:
Does that mean that this is the "removed" version? Or still not pushed?

Reply:
There was a bug in my code that's why I couldn't reproduce anything. It is needed after all.